### PR TITLE
Update The Sims 3 (use Wine stable)

### PR DESCRIPTION
**Pull request by Zemogiter**
Previous used version is now crashing the Sims3LauncherW.exe making it impossible to play. Tried other versions and settled down on this one.

### CODE REVIEW

File: Applications/Games/The Sims 3/Local/script.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ var installerImplementation = {
             .author("Zemogiter")
             .category("Games")
             .executable("Sims3Launcher.exe", ["xgamma -gamma 1"])
-            .wineVersion("3.19")
+            .wineVersion("4.0.1")
```
Comment (plata) [inline Applications/Games/The Sims 3/Local/script.js]: Wouldn't it make sense to use the default (latest stable)?

Comment (Zemogiter) [reply Applications/Games/The Sims 3/Local/script.js]: Maybe but what if they release a new stable 3 months from now and the script stops working?

Comment (plata) [reply Applications/Games/The Sims 3/Local/script.js]: That can always happen. On the other hand, with this approach, somebody always must update the versions manually which is worse in my opinion.

Comment (Zemogiter) [reply Applications/Games/The Sims 3/Local/script.js]: You have a point. I guess lack of action is worse than any action.

**Pull request merged**